components/Common/Select/index.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+import Select from '.';
+
+type Story = StoryObj<typeof Select>;
+type Meta = MetaObj<typeof Select>;
+
+export const Default: Story = {
+  args: {
+    values: ['v20.8.0', 'v19.9.0', 'v18.18.0', 'v17.9.1', 'v16.20.2'],
+    defaultValue: 'v16.20.2',
+    label: 'Node.js version',
+  },
+};
+
+export const Withouth_Label: Story = {
```

Comment:
```suggestion
export const Without_Label: Story = {
```

Reply:
Actually good point, @canerakdas we usually don't use `_` on Story names. Storybook already does the separation by itself. So this story can simply be named `WithoutLabel` (can you update this story name and the others?)

Reply:
My bad, I updated the stories. Also, I have updated the component imports inside the story.